WebContent/plugins/ajax.js

```diff
@@ -1,118 +0,0 @@
-$(document).ready(function(){
-	console.log("hi");
-	$.ajax({
-        url: "././indexOfStudent",
-        method: "GET",
-        success: function (result) 
-        {
-            var htmlFiltered = $(result).filter("#getAllStudents");
-            console.log(htmlFiltered);
-            var finalText = htmlFiltered.html();
-            console.log(finalText);
-            console.log("final");
-            $("#content").html(finalText);
-        }
-    });
-});
-function createStudent()
-{
-	console.log("hi create");
-	$.ajax({
-        url: "././createStudent.jsp",

```

Comment:
url: "././createStudent.jsp" ---- > what is that mean ?? 😠 

use **appContextPath** --- read my allImports.jsp file comments